docker/build-apk.sh

```
@@ -61,13 +66,13 @@ function signApk () {
 
     cp $KEYSTORE "${WORK_DIR}"
 
-    response=$(docker run -it --volume "${WORK_DIR}:/mnt" --workdir /mnt ${DOCKER_IMAGE} bash -x -c \
-      "apksigner sign --ks ${KEYSTORE_FILENAME} --ks-pass pass:${KEYSTORE_PASSWORD} ${BUILT_APK_FILE}")
+    response=$(docker run -it --rm -e KS_PASS --volume "${WORK_DIR}:/mnt" --workdir /mnt ${DOCKER_IMAGE} bash -c \
+      "apksigner sign --ks ${KEYSTORE_FILENAME} --ks-pass env:KS_PASS ${BUILT_APK_FILE}")
 
     echo $response
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)